config/default/manager_auth_proxy_patch.yaml

```diff
@@ -24,5 +24,4 @@ spec:
         - "--health-probe-bind-address=:8081"
         - "--metrics-bind-address=127.0.0.1:8080"
         - "--leader-elect"
-        - "--odf-console-port=9001"
-        - "--ibm-console-port=9003"
+        - "--console-port=9001"
```

Comment:
This patch removes ibm-console. If you want to rename the odf-console-port, then it should be separate commit because it is a logically separate change. It can be discussed

* whether this should remain as `--odf-console-port` for explicitness as @jarrpa  requested
* whether the renaming should even be a separate PR

But at the very least it needs to be a separate commit for starters.